include/boost/math/fft/bsl_backend.hpp

```diff
@@ -198,10 +205,16 @@
     }
 
   private:
+    allocator_type alloc;
     std::size_t my_size{};
     bool _has_root=false;
     RingType my_root, my_inverse_root;
   };
+  
+  } // namespace detail
+  
+  template<class RingType, class Allocator_t = std::allocator<RingType> >
+  using bsl_dft = detail::dft< detail::bsl_backend<RingType,Allocator_t> >;
```

Comment:
Notice that in order to accommodate the allocator template I've changed the API. We will come back very soon to review the state of the API to answer the comments on the previous [PR](https://github.com/BoostGSoC21/math/pull/8#issuecomment-869485169).